packages/manager/apps/telecom/src/app/telecom/telephony/svaWallet/kyc/identity/form/form.controller.js

```diff
@@ -134,6 +141,14 @@ export default class KycIdentityFormController {
       companyKind,
     );
 
+    if (
+      this.wallet.company.marketplace &&
+      (!this.wallet.company.websiteUrl ||
+        this.wallet.company.websiteUrl === undefined)
```

Comment:
```suggestion
      !this.wallet.company.websiteUrl
```
It will covered `undefined` case